pkg/utils/common/pods.py

```diff
@@ -128,7 +128,6 @@ def GetTargetPodsWhenTargetPodsENVNotSet(self, podAffPerc , nonChaosPods, chaosD
 		filteredPods = []
 		realPods = []
 		for pod in nonChaosPods.items:
-		
 			parentName, err = annotation.GetParentName(clients, pod, chaosDetails)
```

Comment:
Can we derive parent name only if annoationCheck is true?

Reply:
change
return isPodsAvailable, None --> return isPodsAvailable, ValueError("{} pod is not available in {} namespace".format(podList[index], namespace)
 https://github.com/litmuschaos/litmus-python/pull/23/files#diff-d887f697d1fd33d3db83a05b0285cd627948aa70f5789b6bafee2084d08ab6f1R31